apps/system/utils/fscmd.c

```diff
@@ -1084,10 +1084,26 @@ static int tash_rm(int argc, char **args)
 {
 	char *fullpath;
 	int ret = OK;
+	struct stat st;
 
 	if (args[2]) {
 		fullpath = get_fullpath(args[2]);
 		if (strncmp(args[1], "-r", strlen(args[1])) == 0) {
+			/* The request is to delete an entire directory recursively
+			   Check if the entry requested is not a direcotyr and report
```

Comment:
Can you please check the spelling of directory ?

Reply:
Rectified, thanks